### ISSUE TITLE
Improve feedback for MIDI CC actions

### ISSUE THREAD

**Issue by helgoboss** [helgoboss/realearn#73]
Right now action feedback is only supported for toggle actions but we might be able to make it work for MIDI CC actions.

**Comment by helgoboss** [comment]
In upcoming ReaLearn v1.12.0-pre9, feedback for MIDI/CC actions should work if you use the latest REAPER prerelease (>= 6.19+dev1226, the one with "hookpostcommand2").

As expected, it comes with some inherent limitations that are related to the fact that a) REAPER itself doesn't necessarily use actions to invoke its own functions and b) REAPER MIDI/CC actions don't have the concept of a "current value" (unlike e.g. toggle actions or FX parameters).

These are the limitations:

1.
In most cases, feedback will not work when changing the value in REAPER directly (e.g. in your case if you change metronome volume via REAPER UI).

2.
It will only work for actions that support some kind of absolute value range (usually the case for all non-relative MIDI/CC actions, e.g. your metronome value action).

3.
When the action is invoked via ReaLearn, the feedback will only work if "Invoke" is "Trigger" or "Absolute". It won't work with "Relative".

4.
When the action is invoked from ReaScript or other extensions, it will only work if the invocation was done via "KBD_OnMainActionEx()" and an absolute value change.

5.
When the action is invoked via a native REAPER action mapping, it will only work if the invocation is done using absolute MIDI CC (not relative).


Bottom line: Only works for absolute actions!
